### PR TITLE
Social-card

### PR DESCRIPTION
**Pull request by escapemanuele**
What should it be the default image? For now I used your selfie (swizec.png)

### CODE REVIEW

File: src/components/head.js [hunk 1]
```diff
@@ -23,10 +24,14 @@ export default (props) => {
     .filter(Boolean)
     .join(" | ")
   const description =
-    (frontmatter && frontmatter.description) || props.description
+    (frontmatter && frontmatter.description) || props.description || "Swizec helps you become a better frontend engineer with books, articles, talks, and workshops";
 
-  const image = props.image || getSocialCard(frontmatter)
-  const url = `${props.path}`
+  const socialImage = getSocialCard(frontmatter);
+  const image = `https://swizec.com${socialImage || defaultHero}`;
```
Comment (Swizec) [inline src/components/head.js:30]: Does this have any implications for how we end up hosting?

File: src/components/head.js [hunk 2]
```diff
@@ -23,10 +24,14 @@ export default (props) => {
...
+  // const url = `${props.path}`
+  const url = `https://swizec.com${
+    props.pageName !== undefined ? `/${props.pageName}` : ""
```
Comment (Swizec) [inline src/components/head.js:33]: In what case would the pageName be undefined?

File: src/components/head.js [hunk 3]
```diff
@@ -23,10 +24,14 @@ export default (props) => {
     .filter(Boolean)
     .join(" | ")
   const description =
-    (frontmatter && frontmatter.description) || props.description
+    (frontmatter && frontmatter.description) || props.description || "Swizec helps you become a better frontend engineer with books, articles, talks, and workshops";
```
Comment (Swizec) [inline src/components/head.js:27]: Can we update code like this to conditional chaining? I think it works with the pipeline we’ve got

```javascript
frontmatter?.description || props.description || “Default description”
```

File: src/components/head.js (hunk 2 above)
Comment (escapemanuele) [reply src/components/head.js:33]: I changed it to props.path. pageName was always undefined (so the url was always https://swizec.com/)

File: src/components/head.js (hunk 1 above)
Comment (escapemanuele) [reply src/components/head.js:31]: What implications are you thinking about?
Anyway I tried it with Netlify and with Gatsby Cloud and it's working

Comment (Swizec) [reply src/components/head.js:31]: Just in terms of URL structure we’re allowed to use. Should be fine if you tested it :)

**Pull request merged**